Update .babylonFileFormat.md

bvaisman-GitHub opened the pull request:
Add SpriteManagers and Sprites to the documentation

bvaisman-GitHub committed:
Update .babylonFileFormat.md

Add SpriteManagers and Sprites

bjsplat commented:
snapshot for this PR available at:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/DOCS/refs/pull/1394/merge

bvaisman-GitHub committed:
Update .babylonFileFormat.md

I've changed the description of the SpriteManager and linked the texture item to JSON Texture

bvaisman-GitHub commented:
It's done, thank you

bjsplat commented:
snapshot for this PR available at:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/DOCS/refs/pull/1394/merge
